validation/src/block_production.rs

```diff
@@ -222,11 +222,10 @@ impl<Client, TxPool, Backend> consensus::Proposer<Block> for Proposer<Client, Tx
 
 			Delay::new(enough_candidates).await;
 
-			tokio_executor::blocking::run(move || {
+			tokio::task::block_in_place(move || {
```

Comment:
@tomaka is this what we want here?

Reply:
Not sure but I don't think so. `tokio::blocking` signals that the given task is blocking and makes sure the runtime runs the task in a separate threadpool.